src/ansiblelint/schemas/ansible-lint-config.json

```diff
@@ -122,60 +122,7 @@
         "$ref": "#/$defs/rule"
       },
       "propertyNames": {
-        "enum": [
-          "command-instead-of-module",
-          "command-instead-of-shell",
-          "deprecated-bare-vars",
-          "deprecated-command-syntax",
-          "deprecated-local-action",
-          "deprecated-module",
-          "empty-string-compare",
-          "fqcn",
-          "fqcn[action-core]",
-          "fqcn[action-redirect]",
-          "fqcn[action]",
-          "galaxy",
-          "galaxy[no-changelog]",
-          "galaxy[tags]",
-          "galaxy[version-incorrect]",
-          "galaxy[version-missing]",
-          "ignore-errors",
-          "inline-env-var",
-          "internal-error",
-          "jinja",
-          "key-order",
-          "latest",
-          "literal-compare",
-          "load-failure",
-          "meta-incorrect",
-          "meta-no-info",
-          "meta-no-tags",
-          "meta-runtime",
-          "meta-video-links",
-          "name",
-          "no-changed-when",
-          "no-handler",
-          "no-jinja-when",
-          "no-log-password",
-          "loop-var-prefix",
-          "no-prompting",
-          "no-relative-paths",
-          "no-same-owner",
-          "no-tabs",
-          "only-builtins",
-          "package-latest",
-          "parser-error",
-          "partial-become",
-          "playbook-extension",
-          "risky-file-permissions",
-          "risky-octal",
-          "risky-shell-pipe",
-          "role-name",
-          "schema",
-          "syntax-check",
-          "var-naming",
-          "yaml"
-        ]
+        "pattern": "^[a-z0-9-]+$"
```

Comment:
`[` and `]` are also part of rule names, e.g. `fqcn[action-core]`, there are none with numbers though.

Reply:
Right, I updated the pattern. Thanks!